### ISSUE TITLE
[feature] allow users to disabled site favicons

### ISSUE THREAD

**Issue by masukomi** [masukomi/backup_brain#26]
Some people just don't want them.

Other people may be concerned about the privacy implications of sending the domain names from their bookmarks to Google.

I'm not sure how google would know who to associate those domains with, and they probably already tracked the person when they visited 99% of them, but... it's reasonable to not want to give that data to google.

Prerequisite: Issue #25  - settings page

**Comment by masukomi** [comment]
done. now available in `next` branch
